src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -80,65 +71,11 @@ public void setInternshipTasks(List<InternshipTask> internshipTasks) {
      */
     public void resetData(ReadOnlyAddressBook newData) {
         requireNonNull(newData);
-        setPersons(newData.getPersonList());
         setAssignments(newData.getAssignmentList());
         setInternshipRoles(newData.getInternshipRoleList());
         setInternshipTasks(newData.getInternshipTaskList());
     }
 
-
-    //// person-level operations
-
-    /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
-     */
-    public boolean hasPerson(Person person) {
-        requireNonNull(person);
-        return persons.contains(person);
-    }
-
-    /**
-     * Adds a person to the address book.
-     * The person must not already exist in the address book.
-     */
-    public void addPerson(Person p) {
-        persons.add(p);
-    }
-
-    /**
-     * Replaces the given person {@code target} in the list with {@code editedPerson}.
-     * {@code target} must exist in the address book.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
-     */
-    public void setPerson(Person target, Person editedPerson) {
-        requireNonNull(editedPerson);
-
-        persons.set(target, editedPerson);
-    }
-
-    /**
-     * Removes {@code key} from this {@code AddressBook}.
-     * {@code key} must exist in the address book.
-     */
-    public void removePerson(Person key) {
-        persons.remove(key);
-    }
-
-    //// util methods
-
-    @Override
-    public String toString() {
-        return new ToStringBuilder(this)
-                .add("persons", persons)
-                .toString();
-    }
```

Comment:
I dont think this toString method shld be removed, just update it with the current lists